app/controllers/students_controller.rb

```diff
@@ -7,18 +7,17 @@ class StudentsController < ApplicationController
 
   def authorize!
     student = Student.find(params[:id])
-    educator = current_educator
-    raise Exceptions::EducatorNotAuthorized unless educator.is_authorized_for_student(student)
+    raise Exceptions::EducatorNotAuthorized unless current_educator.is_authorized_for_student(student)
   end
 
-  def show
+  def show(restricted_notes: false)
```

Comment:
I don't think this will work for Rails controller actions like it would for a normal Ruby method - Rails owns the `params` object and injects it into the method scope.  You can use `params.fetch(:restricted_notes, false)` to do this though.